samples/cpp/matchmethod_orb_akaze_brisk.cpp

```diff
@@ -0,0 +1,89 @@
+#include <opencv2/opencv.hpp>
+#include <vector>
+#include <iostream>
+
+using namespace std;
+using namespace cv;
+
+
+int main(void)
+{
+    vector<String> typeAlgoMatch;
+    typeAlgoMatch.push_back("BruteForce");
+    typeAlgoMatch.push_back("BruteForce-Hamming");
+    typeAlgoMatch.push_back("BruteForce-Hamming(2)");
+
+    vector<String> typeDesc;
+    typeDesc.push_back("AKAZE");
+    typeDesc.push_back("ORB");
+    typeDesc.push_back("BRISK");
+
+    String  dataFolder("../data/");
+    vector<String> fileName;
+    fileName.push_back("basketball1.png");
```

Comment:
Might want to leave a notice here that these images can be found in the data folder under the samples